packages/arb-token-bridge-ui/src/hooks/CCTP/useCCTP.ts

```diff
@@ -126,7 +126,8 @@ export function useCCTP({ sourceChainId }: UseCCTPParams) {
   const fetchAttestation = useCallback(
     async (attestationHash: `0x${string}`) => {
       const response = await fetch(
-        `${attestationApiUrl}/attestations/${attestationHash}`
+        `${attestationApiUrl}/attestations/${attestationHash}`,
```

Comment:
i'm wondering why we don't add `/v1/` as shown on Circle's API ref
https://developers.circle.com/stablecoin/reference/getattestation

`https://iris-api-sandbox.circle.com/v1/attestations/{messageHash}`

probably unrelated since on their demo page they don't have `/v1/`
https://developers.circle.com/stablecoin/docs/cctp-usdc-transfer-quickstart

but better find out which one to use

Reply:
added `v1`